### DISCUSSION TITLE
Backlights plugged in/battery

### DISCUSSION THREAD

**Discussion by zso0** [seerge/g-helper#2679] [General]
Hey, 
The armoury crate has a feature that controls the color and mode of the keyboard backlight depending on whether it's running on battery or connected to the power supply, and it also controls whether the LED strip and the lid LED are lit or not. Could this feature be added to ghelper?

**Comment by seerge** [comment]
@zso0 hello, currently app remembers backlight brightness levels for plugged in / on battery cases and automatically switches when you plug / unplug.

I don't want to overcomplicate things and logic in the app, and remember individual brightness for each separate element :)
